pkg/cmd/root_test.go

```diff
@@ -149,9 +149,9 @@ func TestGetEnvoyManifest(t *testing.T) {
 
 	tests := []testCase{ // we don't test default as that depends on the runtime env
 		{
-			name:     "default is https://tetrate.bintray.com/getenvoy/manifest.json",
+			name:     "default is https://dl.cloudsmith.io/public/tetrate/getenvoy/raw/files/manifest.json",
```

Comment:
curious.. wouldn't this be a good time to use DNS we control instead of having to wonder about a similar future of another SAAS? If this is just manifest json, even GH pages branch might be ok...

Reply:
yah good point, @oktocat  will be setting up so we can use `dl.getenvoy.io` here, just surfacing all the changes needed for the migration for now and stashing these changes in draft mode